utility/src/main/java/org/oppia/util/data/URlImageParser.kt

```diff
@@ -0,0 +1,42 @@
+package org.oppia.util.data
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.graphics.drawable.LevelListDrawable
+import android.text.Html
+import android.widget.TextView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
+import com.bumptech.glide.request.RequestOptions
+import com.bumptech.glide.request.target.SimpleTarget
+import java.net.URL
+
+/** */
+open class URLImageParser(internal var mTv: TextView, internal var mContext: Context) : Html.ImageGetter {
+
+    override fun getDrawable(source: String): Drawable {
+        val drawable = LevelListDrawable()
+        Glide.with(mContext)
+                .asBitmap()
+                .load(URL(source))
+                .apply( RequestOptions().override(100, 100)) //This is important
```

Comment:
Please provide context on why this is important. I actually don't understand this line--why do we want to override width/height to be 100x100?

Reply:
I was testing it and left it after testing. Its not important. I will remove it